tests/bundle/View/SystemInfoViewBuilderTest.php

```diff
@@ -6,31 +6,34 @@
  */
 namespace Ibexa\Tests\Bundle\SystemInfo\View;
 
+use Ibexa\Bundle\SystemInfo\SystemInfo\Collector\SystemInfoCollector;
+use Ibexa\Bundle\SystemInfo\SystemInfo\SystemInfoCollectorRegistry;
 use Ibexa\Bundle\SystemInfo\SystemInfo\Value\SystemInfo;
 use Ibexa\Bundle\SystemInfo\View\SystemInfoViewBuilder;
+use Ibexa\Core\MVC\Symfony\View\Configurator;
 use PHPUnit\Framework\TestCase;
 
 class SystemInfoViewBuilderTest extends TestCase
 {
-    private $configuratorMock;
+    private Configurator $configuratorMock;
```

Comment:
This should be more specific to what actually is inside, and fix on of the still reported PHPStan issues:
```suggestion
    /** @var \PHPUnit\Framework\MockObject\MockObject&\Ibexa\Core\MVC\Symfony\View\Configurator */
    private Configurator $configuratorMock;
```

Similar thing can be done for registry and collector mocks.

Reply:
yeah, I wanted to do proper union type in main. I can add this atm as well tho.